lisp/transient.el

```diff
@@ -512,6 +512,16 @@ slot is non-nil."
    (argument-regexp  :initarg :argument-regexp))
   "Class used for sets of mutually exclusive command-line switches.")
 
+(cl-defmethod initialize-instance ((obj transient-switches) slots)
+  (cl-call-next-method)
+  (-let [(&plist :argument-regexp :choices :argument-format) slots]
+    (unless argument-regexp
+      (setf (oref obj argument-regexp)
```

Comment:
For consistency with other code, use `oset` instead.

Reply:
All modfication have done, plz check